app/forms/mappings/AddressMappings.scala

```diff
@@ -45,6 +45,7 @@ trait AddressMappings extends Mappings {
   protected def postcodeOrCountryMapping(isUKAddress: Boolean): (String, Mapping[String]) = {
     if (isUKAddress) {
       "postCode" -> text("error.required.postcode")
+        .transform[String](_.trim,identity)
```

Comment:
There is a function that has just gone into main that you can use here:

https://github.com/hmrc/amls-frontend/blob/main/app/forms/mappings/Mappings.scala#L53

Reply:
required changes made :)